tests/e2e/backup_restore_suite_test.go

```diff
@@ -47,6 +47,7 @@ var _ = Describe("AWS backup restore tests", func() {
 
 	parksAppReady := VerificationFunction(func(ocClient client.Client, namespace string) error {
 		Eventually(IsDCReady(ocClient, "parks-app", "restify"), timeoutMultiplier*time.Minute*10, time.Second*10).Should(BeTrue())
+		// err := VerifyBackUpRestoreData(artifact_dir, namespace, "restify", "parks-app") // TODO: VERIFY PARKS APP DATA
```

Comment:
thanks... looking at the parks app..  the last commit was in 2016.  I'm a little suspicious of this app. https://github.com/ryanj/restify-mongodb-parks

Reply:
I had added the issue in the description although we could verify the data if we add a wait function.